### ISSUE TITLE
Options with `true` for default value end up being `false` by default

### ISSUE THREAD

**Issue by TooTallNate** [leo/args#71]
```js
const args = require('args')

args
  .option('logs', 'Prints the build logs', true, Boolean)

console.error(args.parse(process.argv))
// { l: false, logs: false }
```

Expected `false` to be `true`.

**Comment by leo** [comment]
Oh, that's correct. Should be `true`!

**Comment by leo** [comment]
@ntwcklng Wanna tackle this? I'd love to add you as a collaborator, btw.

**Comment by ntwcklng** [comment]
@leo yep, i'm on it! Collaborator sounds nice 😎

**Comment by leo** [comment]
@TooTallNate Fixed in [2.4.1](https://github.com/leo/args/releases/tag/2.4.1)!
